src/style.css

```diff
@@ -1 +1,105 @@
-/* add styles here */
+/* stylelint-disable font-family-name-quotes */
+
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
```

Comment:
there is no need to write font-style: normal; cause it works by default

please delete this rule in other places as well